ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master__okd-scos.yaml

```
@@ -0,0 +1,45 @@
+base_images:
+  ocp_4.16_base-rhel9:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+  origin_scos-4.16_base-stream9:
+    name: scos-4.16
+    namespace: origin
+    tag: base-stream9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+    origin_scos-4.16_base-stream9:
+      as:
+      - registry.ci.openshift.org/origin/4.16:base-stream9
+  to: ptp
+promotion:
+  to:
+  - name: ptp-4.17
+    namespace: origin
+releases:
+  latest:
+    integration:
+      include_built_images: true
+      name: scos-4.17
```

```suggestion
      name: scos-4.16
```
master is still 4.17 at this time

---

same elsewhere below

---

yea master is 4.17 right? so the promotion is correct to scos-4.17? My concern is if I do this then both my master/main branch config and 4.16 branch config will promote to ptp-4.16 causing clashes, which would fail tests..